README.md

```diff
@@ -47,41 +47,35 @@ How to build your own jQuery
 
 First, [clone the jQuery git repo](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository).
 
-Then, enter the jquery directory and run the build script:
+Then, enter the jquery directory, install dependencies, and run the build script:
+
 ```bash
-cd jquery && npm run build
+cd jquery
+npm run install
+npm run build
 ```
-The built version of jQuery will be put in the `dist/` subdirectory, along with the minified copy and associated map file.
 
-If you want to create custom build or help with jQuery development, it would be better to install [grunt command line interface](https://github.com/gruntjs/grunt-cli) as a global package:
+The built version of jQuery will be placed in the `dist/` directory, along with a minified copy and associated map file.
```

Comment:
It'd be better to point out here now that we generate multiple files now and some of them are in `dist-module`.

Reply:
`npm run build` only generates one file and it is always placed in the `dist/` directory. `npm run build-all-variants` builds all of our release files, which passes `dir: 'dist-module'` to the build script for the module files.

That said, I've added a section on how to use `npm run build-all-variants` and its purpose.